packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -233,10 +233,28 @@ SparkLineChart.propTypes = {
    * Data to plot.
    */
   data: PropTypes.arrayOf(PropTypes.number).isRequired,
+  /**
+   * An array of object that can be used to populate series and axis data using there `dataKey` property.
+   */
   dataset: PropTypes.arrayOf(PropTypes.object),
   desc: PropTypes.string,
+  /**
+   * If true, the charts will not listen to the mouse move event.
+   * It might breaks interactive features, but save some computation power.
+   * @default false
+   */
```

Comment:
```suggestion
  /**
   * If `true`, the charts will not listen to the mouse move event.
   * It might break interactive features, but is improving performance.
   * @default false
   */
```